Examples/MAX78002/CNN/cifar-100-effnet2/softmax.c

```diff
@@ -16,9 +16,38 @@
  * limitations under the License.
  */
```

Comment:
Does this file need Maxim Copyright Notice? It has already Arm Limited Copyright Notice.